data/src/main/scala/ch.epfl.scala.index.data/github/GithubDownload.scala

```diff
@@ -387,16 +469,51 @@ class GithubDownload(paths: DataPaths,
   }
 
   /**
+    * get the issues query used by the Github GraphQL API
+    *
+    * @return
+    */
+  private def issuesQuery(project: Project): JsObject = {
+
+    val query =
+      """
+        query($owner:String!, $name:String!, $beginnerLabel:String!) {
+          repository(owner: $owner, name: $name) {
+            issues(last:5, states:OPEN, labels:[$beginnerLabel]) {
+              nodes {
+                number
+                title
+                bodyText
+                url
+              }
+            }
+          }
+        }
+      """
+    val beginnerIssuesLabel = project.github match {
+      case Some(github) => github.beginnerIssuesLabel.getOrElse("")
+      case None => ""
+    }
+    Json.obj(
+      "query" -> query,
+      "variables" ->
+        s"""{ "owner": "${project.organization}",
+           |"name": "${project.repository}",
+           |"beginnerLabel": "$beginnerIssuesLabel" }""".stripMargin)
+  }
+
+  /**
     * process all downloads
     */
   def run(): Unit = {
 
     download[GithubRepo, Unit]("Downloading Topics",
                                githubRepos,
-                               githubGraphqlUrl,
+                               null,
```

Comment:
[Done] (https://github.com/scalacenter/scaladex/pull/432/commits/0d0e08ce6dbd290c746345aa8c56493a424dcd08#diff-9b0b9f250047a450506bda2259c11537), added 2 download methods, 1 for normal downloads and 1 for graphql downloads